pkgs/top-level/python-packages.nix

```diff
@@ -5810,6 +5816,10 @@ in {
 
   paperwork-backend = callPackage ../applications/office/paperwork/backend.nix { };
 
+  papis = callPackage ../applications/office/papis { };
+
+  papis-python-rofi = callPackage ../development/python-modules/papis-python-rofi { };
```

Comment:
python-rofi

Reply:
Ah, I see. papis-python-rofi seems to be a fork of python-rofi. Then it's all right like that,